TF_Tooling_Web/src/Cursors/SvgCursor.ts

```diff
@@ -193,4 +194,35 @@ export class SVGCursor extends TouchlessCursor {
         }
         return 0;
     }
+
+    // Function: SetDefaultColors
+    // Used to reset the SVGCursor to it's default styling
+    ResetToDefaultColors() {
+        this.cursor?.setAttribute('fill', this.isDarkCursor ? 'black' : 'white');
+        this.cursor?.removeAttribute('stroke-width');
+        this.cursor?.removeAttribute('stroke');
+        this.cursorRing.setAttribute('stroke', this.isDarkCursor ? 'black' : 'white');
+    }
+
+    // Function: SetColor
+    // Used to set a part of the SVGCursor to a specific color
+    // Takes a number to select the cursorPart where
+    //      0 = center fill
+    //      1 = ring fill
+    //      2 = center border
```

Comment:
👀 Could these numbers be a union type e.g.
`type CursorPart = 'centreFill' | 'centreBorder' | 'ringFill';`
Rather than magic numbers?